website/docs/installation/storage-s3.md

```diff
@@ -79,10 +79,18 @@ AUTHENTIK_STORAGE__MEDIA__S3__ENDPOINT=https://s3.provider
 AUTHENTIK_STORAGE__MEDIA__S3__CUSTOM_DOMAIN=s3.provider/authentik-media
 ```
 
+If you are using Backblaze B2 as your S3 provider, add the following:
```

Comment:
Let's add another remark stating that it's only necessary if the bucket itself is already private.